l2tdevtools/build_helpers/dpkg.py

```diff
@@ -255,10 +255,12 @@ def _GetBuildHostDistribution(self):
       str: Debian/Ubuntu distribution name or None if the value could not
           be determined.
     """
-    lsb_release_path = '/etc/lsb-release'
+    distribution_name = None
 
-    lsb_release = {}
+    lsb_release_path = '/etc/lsb-release'
```

Comment:
Please add comments here to clarify the intent of these two similar blocks of code for debian/ubuntu. Also consider splitting this into two methods, since two different things are being done here. It would also be a good opportunity to refactor to take a file_object arg, so that these methods can be more easily tested.

Reply:
ack